core/config/docs/chains-solana.toml

```diff
@@ -36,6 +36,11 @@ ComputeUnitPriceDefault = 0 # Default
 FeeBumpPeriod = '3s' # Default
 # BlockHistoryPollPeriod is the rate to poll for blocks in the block history fee estimator
 BlockHistoryPollPeriod = '5s' # Default
+# BlockHistorySize is the number of blocks to take into consideration when using FeeEstimatorMode = 'blockhistory' to determine compute unit price.
+# If set to 1, the compute unit price will be determined by the median of the last block's compute unit prices. 
+# If set N > 1, the compute unit price will be determined by the average of the medians of the last N blocks' compute unit prices.
+# DISCLAIMER: 1:1 ratio between n and RPC calls.
```

Comment:
I think we should maybe also call out that this would be every `BlockHistoryPollPeriod` interval so they can accurately gauge the load on their node. Otherwise, if they think this is just once at startup they could still misconfigure the value.